make_dir.sh

```diff
@@ -0,0 +1,82 @@
+#!/bin/bash
```

Comment:
Herre it would be nice to create a comment of wat the script does, whos the author
Smth like this, https://bashcommands.com/bash-script-header there are many formats